products/views.py

```diff
@@ -1,3 +1,26 @@
-from django.shortcuts import render
+from django.http.response  import JsonResponse
+from django.views          import View
+from django.http.response  import JsonResponse
+from products.models       import Product 
 
-# Create your views here.
+
+class CategoryListView(View):
```

Comment:
작성하신 코드를 보면 카테고리가 아니라 프로덕트 리스트인 것 같은데 맞을까요?
```suggestion
class ProductListView(View):
```